### PR TITLE
Remove Errorable from component names

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/18146

## Testing done
Storybook :eyes: 

## Screenshots
image

### CODE REVIEW

File: packages/formation-react/README.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ From the project root, run the following commands:
...
-- ErrorableTextInput.js
+- Checkbox.js
+- Date.js
+- FileInput.js
+- MonthYear.js
+- NumberInput.js
+- RadioButtons.js
+- Select.js
+- TextArea.js
+- TextInput.js
```
Comment (bkjohnson) [inline packages/formation-react/README.md:36]: This gets reflected in the `Visual Design` sidebar in the gatsby site, right?

File: packages/formation-react/src/components/Checkbox/Checkbox.mdx [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
 ### Code:
 ```javascript
-import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox'
+import Checkbox from '@department-of-veterans-affairs/formation-react/Checkbox'
 
 <div>
-  <ErrorableCheckbox
-    label='Errorable Checkbox'
+  <Checkbox
+    label=' Checkbox'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/Checkbox/Checkbox.mdx:15]:
```suggestion
    label='Checkbox'
```

File: packages/formation-react/src/components/Checkbox/Checkbox.mdx [hunk 3]
```diff
@@ -1,73 +1,73 @@
...
   />
 </div>
 ```
 
 ### Rendered Component
 <div className="site-c-reactcomp__rendered">
-  <ErrorableCheckbox
-    label='Errorable Checkbox'
+  <Checkbox
+    label=' Checkbox'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/Checkbox/Checkbox.mdx:47]:
```suggestion
    label='Checkbox'
```

File: packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 
 ### Code:
 ```javascript
-import ErrorableCheckboxGroup from '@department-of-veterans-affairs/formation-react/ErrorableCheckboxGroup'
+import CheckboxGroup from '@department-of-veterans-affairs/formation-react/CheckboxGroup'
 
-<ErrorableCheckboxGroup
-  label='Errorable Checkbox Group'
+<CheckboxGroup
+  label=' Checkbox Group'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx:14]:
```suggestion
  label='Checkbox Group'
```

File: packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
   onValueChange={(value) => value}
   id='default'
   errorMessage='This is an error message'
-  label='Errorable Checkboxes'
+  label=' Checkboxes'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx:18]:
```suggestion
  label='Checkboxes'
```

File: packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx [hunk 6]
```diff
@@ -37,12 +37,12 @@ import ErrorableCheckboxGroup from '@department-of-veterans-affairs/formation-re
 
 ### Rendered Component
 <div className="site-c-reactcomp__rendered">
-  <ErrorableCheckboxGroup
-    label='Errorable Checkbox Group'
+  <CheckboxGroup
+    label=' Checkbox Group'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx:41]:
```suggestion
    label='Checkbox Group'
```

File: packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx [hunk 7]
```diff
@@ -37,12 +37,12 @@ import ErrorableCheckboxGroup from '@department-of-veterans-affairs/formation-re
...
     onValueChange={(value) => value}
     id='default'
     errorMessage='This is an error message'
-    label='Errorable Checkboxes'
+    label=' Checkboxes'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.mdx:45]:
```suggestion
    label='Checkboxes'
```

File: packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.stories.jsx [hunk 8]
```diff
@@ -13,12 +13,12 @@ export default {
...
 const Template = args => (
   <div style={{ paddingLeft: '1em' }}>
-    <ErrorableCheckboxGroup {...args} />
+    <CheckboxGroup {...args} />
   </div>
 );
 
 const defaultArgs = {
-  label: 'Errorable Checkbox Group',
+  label: ' Checkbox Group',
```
Comment (bkjohnson) [inline packages/formation-react/src/components/CheckboxGroup/CheckboxGroup.stories.jsx:21]:
```suggestion
  label: 'Checkbox Group',
```

File: packages/formation-react/src/components/Date/Date.mdx [hunk 9]
```diff
@@ -32,13 +32,13 @@ this.state = {
...
-import ErrorableDate from '@department-of-veterans-affairs/formation-react/ErrorableDate'
+import Date from '@department-of-veterans-affairs/formation-react/Date'
 
-<ErrorableDate
+<Date
   onValueChange={date => this.setState({ date })}
   required={true}
   label='Please tell us a date'
-  name='Errorable Date'
+  name=' Date'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/Date/Date.mdx:41]:
```suggestion
  name='Date'
```

File: packages/formation-react/src/components/Date/Date.mdx [hunk 10]
```diff
@@ -70,11 +70,11 @@ export class RenderedComponent extends React.Component {
   render () {
     return (
       <div className="site-c-reactcomp__rendered">
-        <ErrorableDate
+        <Date
           onValueChange={(date) => this.setState({date})}
           required={true}
           label='Please tell us a date'
-          name='Errorable Date'
+          name=' Date'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/Date/Date.mdx:77]:
```suggestion
          name='Date'
```

File: packages/formation-react/src/components/Date/Date.stories.jsx [hunk 11]
```diff
@@ -20,7 +20,7 @@ const Template = args => {
 const defaultArgs = {
   required: true,
   label: 'Please tell us a date',
-  name: 'Errorable Date',
+  name: ' Date',
```
Comment (bkjohnson) [inline packages/formation-react/src/components/Date/Date.stories.jsx:23]:
```suggestion
  name: 'Date',
```

File: packages/formation-react/src/components/RadioButtons/RadioButtons.mdx [hunk 12]
```diff
@@ -20,14 +20,14 @@ state = {
...
+import RadioButtons from '@department-of-veterans-affairs/formation-react/RadioButtons'
 
-<ErrorableRadioButtons
+<RadioButtons
   onValueChange={(value)=> this.setState({ value })}
   id='defaultId'
   name='defaultName'
   errorMessage='Radio Button errorMessage'
-  label='Errorable Radio Buttons'
+  label=' Radio Buttons'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/RadioButtons/RadioButtons.mdx:30]:
```suggestion
  label='Radio Buttons'
```

File: packages/formation-react/src/components/RadioButtons/RadioButtons.mdx [hunk 13]
```diff
@@ -58,12 +58,12 @@ export class RenderedComponent extends React.Component {
...
     return (
       <div className="site-c-reactcomp__rendered">
-        <ErrorableRadioButtons
+        <RadioButtons
           onValueChange={(value)=> this.setState({ value })}
           id='defaultId'
           name='defaultName'
           errorMessage='Radio Button errorMessage'
-          label='Errorable Radio Buttons'
+          label=' Radio Buttons'
```
Comment (bkjohnson) [inline packages/formation-react/src/components/RadioButtons/RadioButtons.mdx:66]:
```suggestion
          label='Radio Buttons'
```

**Review by bkjohnson — APPROVED** [review]
:pick: :pick: 

For posterity can you put the shell script(s) you used in the PR description? I might like to use something like that in the future and I know I'll never remember it :sweat_smile:

File: packages/formation-react/README.md (hunk 1 above)
Comment (cvalarida) [reply packages/formation-react/README.md:36]: Ooh, good question. I doubt it. :grimacing:

**Comment by cvalarida** [comment]
Ooh, yeah. That's actually why I started that Slack thread tracking these commands in the first place! :sweat_smile: 

Rename all the files in `formation-react`:
1) ```sh
   find . -type d -name "Errorable*" -exec bash -c 'x="{}"; mv "$x" "$(sed -E 's/Errorable//g' <(echo "$x"))"' \;
   ```
    - Gotta do the directories first and then the files
    - When I tried to do them all at once it couldn't find the files because their parent directory was renamed :roll_eyes: 
2) ```sh
   find . -type f -name "Errorable*" -exec bash -c 'x="{}"; mv "$x" "$(sed -E 's/Errorable//g' <(echo "$x"))"' \;
   ```

Remove every instance of `Errorable` in the file contents:
```sh
rg "Errorable" --files-with-matches | xargs -I {} sed -Ei 's/Errorable//g' {}
```

**Pull request merged**